docs/source/_templates/layout.html

```diff
@@ -31,6 +31,23 @@
     {% include "searchbox.html" %}
 {% endblock %}
 
+{%- block content %}
+{{ super() }}
+<script>
+
+//const re = new RegExp("/_[a-zA-Z0-9_]*.html")
+//const match = window.location.href.match(re)
+
+var match      = window.location.href.match(/\/_[a-zA-Z0-9_]*.html|_dynamo/gi);
+var url  = window.location.href.lastIndexOf(match[match.length-1]);
+
+if (url)
+  {
+    var div = '<div class="admonition note"><p class="admonition-title">Note</p><p><i class="fa fa-exclamation-circle" aria-hidden="true">&nbsp</i> This page describes a private API. No support is provided for private APIs.</p></div>'
```

Comment:
How about something along the following lines.
```suggestion
    var div = '<div class="admonition note"><p class="admonition-title">Note</p><p><i class="fa fa-exclamation-circle" aria-hidden="true">&nbsp</i> This is a private API, which is not supposed to be used outside of PyTorch codebase, and can be changed or removed without any notice.</p></div>'
```